agimus_controller_ros/agimus_controller_ros/simple_trajectory_publisher.py

```diff
@@ -79,8 +80,8 @@ def __init__(self):
             ),
         )
         self.state_subscriber = self.create_subscription(
-            JointState,
-            "joint_states",
+            Sensor,
+            "sensor",
```

Comment:
Why is it sensor now ?

Reply:
This comes from the linear-feedback-controller hence we wait until the LFC is active hence we can safely assume the robot is static in it's initial pose.

The sensor contained base + joint information from sensors.